infrastructure/dpladm/bin/sync-site.sh

```diff
@@ -34,6 +34,19 @@ function getSiteDplCmsRelease {
     return
 }
 
+
+function getWebmasterDplCmsRelease {
+    local wmRelease
+    wmRelease=$(yq eval ".sites.${1}.webmaster-cms-version" "${2}")
+    if [[ "${wmRelease}" == "null" ]]; then
+        echo ""
+        return
+    fi
+
+    echo "${wmRelease}"
+    return
+}
```

Comment:
How will the caller `syncEnvRepo` handle an empty string as a response?

I am no expert in Bash but I wonder if it would be preferable if the function failed entirely if this is missing - or alternately used `releaseTag` with a message explaining that no information about a webmaster specific function could be found.

Reply:
This is a copy of the already tried and tested function that finds the release for `main`. If there is an empty string that's what's returned. It then defaults to the latest image.